editoast/migrations/2024-02-06-171107_remove_features_from_rollingstock/up.sql

```diff
@@ -0,0 +1,7 @@
+
+ALTER TABLE rolling_stock
+DROP COLUMN features,
+ALTER COLUMN supported_signaling_systems DROP DEFAULT,
+ALTER COLUMN supported_signaling_systems TYPE TEXT[] USING TRANSLATE(supported_signaling_systems::jsonb::text, '[]','{}')::TEXT[],
+ADD CONSTRAINT supported_signaling_systems_no_nulls CHECK (array_position(supported_signaling_systems, NULL) IS NULL),
+ALTER COLUMN supported_signaling_systems SET DEFAULT ('{"BAPR", "BAL", "TVM300", "TVM430"}');
```

Comment:
We don't need the constraint and a default value.

```suggestion
ALTER TABLE rolling_stock DROP COLUMN features,
    ALTER COLUMN supported_signaling_systems DROP DEFAULT,
    ALTER COLUMN supported_signaling_systems TYPE TEXT [] USING TRANSLATE(
        supported_signaling_systems::jsonb::text, '[]', '{}')::TEXT [];
```